cli/cli_info.cpp

```diff
@@ -119,6 +120,9 @@ command(int argc, char *argv[])
             return 1;
         }
 
+	auto &properties = p.getProperties();
+	std::string processName = properties.find("process.name")->second;
```

Comment:
Old traces don't have process name (or any properties at all.)  You need to cope with that here.  Perhaps we can pass JSON's `null` in that case.

Otherwise LGTM.

Reply:
do you have some trace in old format laying around for me to test?

Reply:
https://github.com/apitrace/apitrace-tests/tree/master/traces has a few